### PR TITLE
Add support for StringSliceEnvVar flag.

### PR DESCRIPTION
**Pull request by mm4tt**
It will be needed to support multiple masters in cl2.

### CODE REVIEW

**Comment by mm4tt** [comment]
@oxddr, could you please lgtm again? I needed to fix gofmt issue

File: clusterloader2/pkg/flags/types.go [hunk 1]
```diff
@@ -57,6 +58,36 @@ func (s *stringFlagFunc) Type() string {
...
+func (*stringSliceFlagFunc) String() string {
+	return "false"
+}
+
+// Set handles flag value setting.
+func (s *stringSliceFlagFunc) Set(val string) error {
+	if val != "" {
+		*s.valPtr = strings.Split(val, ",")
+	} else {
+		*s.valPtr = nil
```
Comment (krzysied) [inline clusterloader2/pkg/flags/types.go]: Do we need this `else`? Isn't `nil` a default value?

Comment (mm4tt) [reply clusterloader2/pkg/flags/types.go]: The flag supports setting both via env var and via flag. So let's say it's MY_STRING_LIST env or --my_string_list flag.

Let's say the MY_STRING_LIST is set to "1,2,3" in some upstream code that we cannot really change, but we want to set it to empty list via flag override. This else clause allows it, I can empty the list by setting --my_string_list=""

**Comment by oxddr** [comment]
and again :D 

/lgtm

File: clusterloader2/pkg/flags/types.go (hunk 1 above)
Comment (krzysied) [reply clusterloader2/pkg/flags/types.go]: Ok. :+1:

**Pull request merged**